### DISCUSSION TITLE
kmod-inet-diag 如果未安装，能正常用么！？公版openwrt有办法安装么？

### DISCUSSION THREAD

**Discussion by phoenix13023** [vernesong/OpenClash#2517] [Q&A]
最近更新了新版，结果发现依赖库kmod-inet-diag 无法安装成功。

想问下，这个不安装是什么结果？会有什么功能影响么？

如果想ipk安装，是否有什么办法？

**Comment by warcns** [comment]
目前不安装kmod-inet-diag也可以正常使用openclash

**Comment by gitx6** [comment]
因为无法安装 kmod-inet-diag，一直停留在 v0.44.42-beta 不敢升级。

**Reply by warcns** [reply]
我的openwrt也是一直装不上kmod-inet-diag，最新版也一样可以用。
